web-components/test/merch-card.mini-compare.test.html

```diff
@@ -422,16 +422,18 @@ <h5 id="price---abm---creative-cloud-all-apps-with-4tb" slot="heading-m-price"><
                 <p slot="price-commitment" class="price-commitment"></p>
                 <h3 id="creative-cloud-all-apps" slot="heading-m">Creative Cloud All Apps
                 </h3>
-                <p slot="callout-text">AI Assistant add-on available <img src="./img/info-icon.svg"/></p>
-                <p slot="callout-text">
-                    <span>
-                        For a limited time pay <span is="inline-price" data-display-per-unit="false"
-                        data-display-recurrence="true" data-display-tax="false" data-wcs-osi="puf"></span> when you add AI assistant to Acrobat Reader, Standard, or Pro. Ends 6/4.
-                    </span
-                    ><img src="./img/info-icon.svg"/>
-                </p>
                 <h4 id="this-is-promo-text-with-a-link" slot="detail-m">This is promo text <a href="https://adobe.com/"
                         target="_blank">with a link</a></h4>
+                <div slot="callout-text">
+                    <p>AI Assistant add-on available <img src="./img/info-icon.svg"/></p>
```

Comment:
`alt`  text is missing on the `img`

Reply:
I have done some changes, please see if it looks fine now